### ISSUE TITLE
Install location of tidy headers

### ISSUE THREAD

**Issue by geoffmcl** [htacg/tidy-html5#646]
Through working with 'edbrowse' project, note the **install** location of tidy headers **changed** from just `include` as in the `master` release branch, to `include/tidy` in the `next` branch! Why? What reason?

To me tidy headers should be in just the `include` folder, like they were in the last **release**... 

We even renamed some headers to make them less `generic` so they could safely reside in the `include` directory... there seems no reason to push them down into a `tidy` subdirectory...

My FindTidy.cmake modules does have both `include` and `include/tidy` so it seems I noted this some time ago, but I guess did not particularly think about it that much... 

But in edbrowse it is in the above order, so when Karl tried to test the `issue-643` branch, he ran into the problem that it installed in `include/tidy`, so the last `release` master headers were found first, causing an assert problem, and maybe other unseen problems...

Do not have time to fully research this deeply tonight, but hope this was just a `slip` in the `next` CMakeLists.txt, and can be quickly undone... thanks...

**Comment by balthisar** [comment]
Looks like @spk made that change about six months ago or so, and I merged it after minimal discussion. Given the standard header search paths, it didn't affect me, although I pointed out the issue. It doesn't affect Unix builds, so I'm ambivalent, willing to roll it back.

@spk, is there a reason in your issue that the headers _must_ be in a subdirectory? Can we roll this back, or will something else break?

**Comment by geoffmcl** [comment]
@balthisar, thanks for your comment, arrived while preparing this addition, but seems nothing at odds here...

Ok, back after dinner, and found the commit 3333ca7d694, that added this change to the `INCLUDE_INSTALL_DIR`, but can **not** exactly find the reason...

Can understand that maybe `tidy.pc` needs to be installed there, although **not** exactly, but why should this alter where the tidy API headers are installed?

And in fact, in Windows, the `tidy.pc` is installed in `lib\pkgconfig\tidy.pc`, and I can see others there like `libcurl.pc`, `openal.pc`, ... while I can see others, like `freeglut.pc`, and `zlib.pc` are installed in `share\pkgconfig`... 

Do **not** know which is **correct**, the **right** location for a `package config` file... **not** ever used in Windows...

But, again in Windows, that `tidy.pc` contains an entry `includedir=${prefix}/include/tidy` which seems **wrong**...

And it seems I inadvertently signed off on this, as a `*nix` issue, in PR #534, by @spk... I concede I should have paid more attention... sorry...

While I am still **not** adverse to generating and installing a `tidy.pc` file, where ever, this should **not** alter the install location of the tidy header files...

They should be in simply `include`, exactly like the last `release` 5.4, in the `master` branch...

So do **not** suggest a `roll back` of the PR, just some adjustments...

Look forward to a patch, or PR, or feedback, to **fix** this, quickly... thanks...

**Comment by geoffmcl** [comment]
@balthisar, @spk, as far as I can see, this is just a CMakeLists.txt one line change...

```diff
diff --git a/CMakeLists.txt b/CMakeLists.txt
index edd6439..20939f3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -373,7 +373,7 @@ if (NOT BIN_INSTALL_DIR)
 endif ()
 
 if (NOT INCLUDE_INSTALL_DIR)
-    set(INCLUDE_INSTALL_DIR include/${LIB_NAME})
+    set(INCLUDE_INSTALL_DIR include)
 endif ()
 
 
```

Have pushed the change to an `issue-646` branch.

Can this PR #647 be merged to `next`? Thanks...

**Comment by spk** [comment]
Hi @geoffmcl @balthisar , sorry about that. I think I did that to do like other *nix distros.
PR #647 look good to me @geoffmcl thanks for the fix

**Comment by geoffmcl** [comment]
@spk thanks for the quick response... now merged PR #647, and version bumped to 5.5.77...
